ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Setup.hs

```diff
@@ -48,20 +45,17 @@ runTest ::
 runTest schedulerConfig genesisTest schedule makeProperty = do
     (recordingTracer, getTrace) <- recordingTracerTVar
     let tracer = if scDebug schedulerConfig then debugTracer else recordingTracer
+
+    -- FIXME: should also go in 'prettyGenesisTest' (or 'prettyBlockTree')
```

Comment:
Shouldn't this be a `TODO`? We could leave `FIXME` For things that *have* to be fixed before merging something.

Reply:
Done in d96aee77387eccb196522b4e5b3df1b61518440c.